combat-trainer.lic

```diff
@@ -2215,6 +2215,11 @@ class TrainerProcess
     when 'App Bundle'
       retreat
       bput("app my bundle quick", 'You scan', 'Appraise what', 'You cannot appraise', 'You can\'t appraise')
+    when /Summon (?<element>Aether|Air|Earth|Electricity|Fire|Water)/
+      retreat
```

Comment:
> Summoning an domain of your aligned element requires 200 ranks of summoning skill

Do you think we should check for a minimum Summoning skill?

Reply:
Yes

Reply:
Corgar confirmed with a newbie warrior mage that below 200 summoning you can't summon an elemental domain

```
> summon aether domain
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create an aether domain spanning the area.
Try though you may, the ritual seems hopelessly involved.
Roundtime: 10 sec.
```